### ISSUE TITLE
llvm verifier fails on some tests for LLVM 7

### ISSUE THREAD

**Issue by mppf** [chapel-lang/chapel#9576]
I'm seeing these test fail to verify with LLVM 7:

```
arrays/bradc/arrayOfArrayArg-inferred.chpl
arrays/bradc/enumarray2.chpl
arrays/bradc/enumIterate.chpl
arrays/deitz/opaque/test_opaque1.chpl
arrays/deitz/opaque/test_opaque2.chpl
arrays/deitz/opaque/test_opaque3.chpl
arrays/deitz/opaque/test_opaque_remove.chpl
```

**Comment by mppf** [comment]
@dmk42 - FYI, since we'll probably want to upgrade LLVMs at some point.

**Comment by dmk42** [comment]
Thanks, good to know.

**Comment by mppf** [comment]
I'm not seeing this issue anymore with LLVM 8 pre-release so I'll close.

**Comment by dmk42** [comment]
Also, I can't reproduce this with LLVM 7 on chapcs anymore.  (I was going to see if 7.0.1 fixed it, but couldn't make it fail with 7.0.0.)
